bfxapi/rest/bfx_rest.py

```diff
@@ -365,6 +365,21 @@ async def get_funding_credit_history(self, symbol, start, end, limit=25):
         credits = await self.post(endpoint, params=params)
         return [FundingCredit.from_raw_credit(c) for c in credits]
 
+    async def get_ledgers(self, symbol, start, end, limit=25, category=None):
+        """
+        Get all ledgers on account associated with API_KEY - Requires authentication.
+        See category filters here: https://docs.bitfinex.com/reference#rest-auth-ledgers
+        @return Array <models.Ledger>
+        """
+        endpoint = "auth/r/ledgers/{}/hist".format(symbol)
```

Comment:
You can also emit the 'symbols' param  in order to receive data for all symbols, would be nice to accommodate this

Reply:
Added in https://github.com/bitfinexcom/bitfinex-api-py/commit/1812942c5fa7770e3fb8089288950a5b8da2d53f